examples/examples_nodejs_test.go

```diff
@@ -156,55 +148,22 @@ func retryFunc(t *testing.T, fn func() bool) error {
 	}
 }
 
-// isConnected validates that the $connect lambda was triggered successfully by
```

Comment:
I'm wondering whether we should leave the example in if it's too flaky.
It's not the best experience if somebody tries it out and can't get it working (even if the problem is not on the Pulumi end)

Reply:
Yeah you are probably right. I also think websockets is a weird example to pick, we should probably switch the example to just be a normal HTTP api.

Reply:
Good question! I think of examples and tests as serving two distinct purposes. If the test bits (verification) are flaky, but the users can still `cd example/foo && pulumi up` then it still might have value as an example and worth leaving in.

Reply:
https://github.com/pulumi/pulumi-cdk/issues/159